scripts/release-notes.js

```diff
@@ -53,7 +53,9 @@ async function getCommits() {
       const jsonReady = `[${commitsArray
         .map(rawCommitText => {
           return rawCommitText
+            .replace(/	/g, ' ')
             .replace(/"/g, '\\\"')
+            .replace(/'/g, '\\\"')
```

Comment:
Would you please provide samples of problem commits, and how does this change fix them?

Reply:
[1.txt](https://github.com/cawa-93/vite-electron-builder/files/6222236/1.txt)
unfortunately the repository is private, here is the log. problems start on 2 line

Reply:
[1.txt](https://github.com/cawa-93/vite-electron-builder/files/6222265/1.txt)
with problem quotes

Reply:
I still don't see the need to replace single quotes `'` with double quotes `"`.

I tried to reproduce your problem. Without replacing quotes, everything works as expected:
```

    Merge remote-tracking branch 'remote/main' into test-local

    # Conflicts:
    #       .github/FUNDING.yml
    #       .github/workflows/release.yml
    #       README.md
    #       electron-vendors.config.js
    #       package-lock.json
    #       package.json
    #       packages/renderer/vite.config.js
```
```markdows
### Other Changes
- Merge remote-tracking branch 'remote/main' into test-local. (d08582b)
  # Conflicts:
  #  .github/FUNDING.yml
  #  .github/workflows/release.yml
  #  README.md
  #  electron-vendors.config.js
  #  package-lock.json
  #  package.json
  #  packages/renderer/vite.config.js
```

Reply:
ok, cleaned it up, I didn't really understand why I had such a problem.